assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -64,7 +64,7 @@ $z-layers: (
 		'.chart__empty': 1,
 		'.chart__bar-marker': 1,
 		'.chart__bar-section.is-ghost::after': 1,
-		'.billing-history-page .filter-popover-content': 1,
+		'.billing-history-page .filter-popover-content': 23,
```

Comment:
Since we've bumped the value to 23, let's move this down further above `.reader-mobile-sidebar` you can see that this beats the search component which is set at `'.search': 22,`